0.preprocess-sites/1.process-spots.py

```diff
@@ -68,14 +68,14 @@
 project = core_args["project"]
 batch = core_args["batch"]
 batch_dir = core_args["batch_dir"]
+quality_func = core_args["cell_quality"]
```

Comment:
I need to confirm this update

Reply:
Do you want to confirm this before merging pull request?

Reply:
yep, thanks for the reminder. I think the source of confusion yesterday was I thought to make this update but only did 1/2 of the work. The remaining fix (and improving variable naming) is added in d3752c2f99a43bfd2631d5a024b96b8cd0dbe361